### ISSUE TITLE
Remix A & C outputs

### ISSUE THREAD

**Issue by bitfetcher** [dBiz/VCVRack_Modules#23]
Hello,

Great set of modules! 

On the Remix, though, I'm not seeing any output on A or C, and after a quick look at the code, I don't think those outputs are ever assigned in the "step" function? 

I cobbled this together and it seems to work, but, by all means, check me!! It replaces the code following (and including) the assignment of 0.0f to outputs[B_OUTPUT].

Thanks again for the fun modules!!

```
    outputs[A_OUTPUT].value = 0.0f;
    outputs[B_OUTPUT].value = 0.0f;
    outputs[C_OUTPUT].value = 0.0f;

    for (int i = 0; i < 6; i++)
    {
        outputs[i].value = ins[i] * inMults[i];

        lights[CH1_LIGHT + i].setBrightnessSmooth(fmaxf(0.0, inMults[i]));

        outputs[B_OUTPUT].value = outputs[B_OUTPUT].value + outputs[i].value;

        if (i <= 1)
        {
            outputs[A_OUTPUT].value = outputs[A_OUTPUT].value + outputs[i].value;
        }
        else if (i >= 4)
        {
            outputs[C_OUTPUT].value = outputs[C_OUTPUT].value + outputs[i].value;
        }

    outputs[A_OUTPUT].value = crossf(outputs[A_OUTPUT].value * params[LEVEL_PARAM].value, outputs[A_OUTPUT].value * params[LEVEL_PARAM].value*clampf(inputs[LEVEL_INPUT].normalize(10)/10,0.0,1.0),params[CV_LEVEL_PARAM].value);
    outputs[B_OUTPUT].value = crossf(outputs[B_OUTPUT].value * params[LEVEL_PARAM].value, outputs[B_OUTPUT].value * params[LEVEL_PARAM].value*clampf(inputs[LEVEL_INPUT].normalize(10)/10,0.0,1.0),params[CV_LEVEL_PARAM].value);
    outputs[C_OUTPUT].value = crossf(outputs[C_OUTPUT].value * params[LEVEL_PARAM].value, outputs[C_OUTPUT].value * params[LEVEL_PARAM].value*clampf(inputs[LEVEL_INPUT].normalize(10)/10,0.0,1.0),params[CV_LEVEL_PARAM].value);
    }

```

**Comment by dBiz** [comment]
thx ;) yep the A and C output has to be fixed! I'm working now for 0.6 API update and i'll include a fixed version of remix!

**Comment by dBiz** [comment]
Corrected code in 060dev ;)
